### ISSUE TITLE
[User flows and navigating] There are no implemented entry points, or a recommended entry point for your new content or feature is missing.  (03.06.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#50947]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
Authenticated 22-1990e Transfer of Entitlement (TOE) Form Updates

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
There are no implemented entry points, or a recommended entry point for your new content or feature is missing. 

### Issue Details
When I navigate home > education and training > survivor and dependent benefits page, the only transferred benefits url on the page links to the old URL and form, not the one proposed in this review. This does not align with the IA guidance provided by sitewide content & IA. The only entry point that exists is hidden within a wizard that requires multiple clicks to surface an application link. Neither the wizard or "how to apply" page have contextual information regarding education benefit transfer that could help a user confirm that they are in the right place.

### VA.gov Experience Standard
[Category Number 03, Issue Number 06](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.5_AA

### Platform Recommendation
Additional entry points must be added to direct users clearly to this form. 

Additionally, the link to the old form that you intend to maintain for unauthenticated users has a different design but exact match H1 and breadcrumbs (as does the form it links out to) to the new page you propose. This is confusing. The forms differ (the OMB time estimation on the unauthenticated version is significantly higher). Please add contextual information to make the differences clear. If the sole planned difference is that one page is intended for authenticated users, and one is for unauthenticated, the design of these pages should be more in sync. They are currently very different and present confusion. Imagine a scenario where a user begins the new form, saves their application, and leaves the site; they come back tomorrow but accidentally navigate to the old form that is completely different and don't understand what is happening. How can we better direct users and minimize confusion? Please refer back to sitewide content & IA's recommendations for entry points and new static pages to situate this new form appropriately.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by sterkenburgsara** [comment]
@btruongafs this was previously labeled as launch blocking. I have edited the severity of the feedback, and believe it is one step below that, but is still very close to being a launch-blocking issue. Currently in the existing IA, users have to navigate to the "how to apply" page and click a button, then select three separate radio buttons in order to surface the application link. In total, users must make five clicks on the correct page, none of which have any contextual information or keywords regarding "education benefit transfer application." Therefore, this new version of the form is very difficult to find and most users will not land on it. They are significantly more likely to navigate to the older form. I recommend that your team launch after the corresponding sitewide content & IA pages have been put in place to improve navigation with those additional entry points.

As an easy/slight improvement, you could add explanatory text to the first radio button in your modal "applying for a new benefit" (e.g. "applying for a new benefit or applying to use a transferred benefit") to make it clearer to these specific users that they are in the right place, but please consult with sitewide content and IA regarding that recommendation. Many users rely on keyword foraging to decide if they are in the right place, and without any context here, this is a challenging IA.

**Comment by sterkenburgsara** [comment]
Looping you in @bethpotts @mnorthuis @DanielleThierryUSDSVA re: updates on new static pages to support the 22-1990e.

**Comment by sterkenburgsara** [comment]
@btruongafs I discussed this with OCTO's information architect @mnorthuis and new launch-blocking issues will be opened that are related to this ticket but slightly different. I missed this in time for the review and Mikki caught it, but the breadcrumb for this form is not correct, so one ticket is about fixing that. The breadcrumb issue points to the second launch-blocking ticket, which is that no entry point exists on the page that navigational breadcrumbs points users toward (survivor and dependent benefits page). Please come to our office hours anytime or feel free to ping me on Slack with questions. Thank you! // cc @mnorthuis

**Comment by btruongafs** [comment]
@RLHecht @bethpotts @katherine-fung I believe this work has been completed and now tracking for deployment with https://github.com/department-of-veterans-affairs/va.gov-team/issues/50830. Can you confirm if there's anything else to do or can we close out?

**Comment by RLHecht** [comment]
Yep, now that we have the static page in Drupal this can be closed out.
